### PR TITLE
sync aya-tool example with aya-template

### PR DESCRIPTION
**Pull request by dmitris**
_structopt delenda est_

### CODE REVIEW

File: examples/aya-tool/myapp-ebpf/rust-toolchain.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel="nightly-2023-01-10"
+channel = "nightly-2023-01-10"
```
Comment (vadorovsky) [inline examples/aya-tool/myapp-ebpf/rust-toolchain.toml:2]: We don't pin nightly anymore. Are you sure you used the latest aya-template?

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: examples/aya-tool/myapp-ebpf/rust-toolchain.toml (hunk 1 above)
Comment (dmitris) [reply examples/aya-tool/myapp-ebpf/rust-toolchain.toml]: unpinned, also in `.github/workflows/ci.yml`.  "sync with template" is figuratively speaking in the sense to reduce the differences I noted - I don't know a way to "mechanically" sync the two.

File: examples/aya-tool/xtask/src/main.rs [hunk 2]
```diff
@@ -3,21 +3,21 @@ mod run;
 
 use std::process::exit;
 
-use structopt::StructOpt;
-#[derive(StructOpt)]
+use clap::Parser;
+#[derive(Debug, Parser)]
```
Comment (vadorovsky) [inline examples/aya-tool/xtask/src/main.rs:7]: Wait, is it generated like that? Without an empty line in between the import and `derive`? :scream: 

If so, we need to fix that. I remember you got it working. Did we forget to fix that properly also in xtask?

File: examples/lsm-nice/xtask/src/main.rs [hunk 3]
```diff
@@ -3,21 +3,21 @@ mod run;
 
 use std::process::exit;
 
-use structopt::StructOpt;
-#[derive(StructOpt)]
+use clap::Parser;
+#[derive(Debug, Parser)]
```
Comment (vadorovsky) [inline examples/lsm-nice/xtask/src/main.rs:7]: same here

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: examples/aya-tool/xtask/src/main.rs (hunk 2 above)
Comment (vadorovsky) [reply examples/aya-tool/xtask/src/main.rs:7]: Nah, it looks fine in the template

https://github.com/aya-rs/aya-template/blob/main/xtask/src/main.rs#L7

Please fix it here

Comment (dmitris) [reply examples/aya-tool/xtask/src/main.rs:8]: done

**Review by vadorovsky — APPROVED** [review]

**Pull request merged**